observability/pillars/dashboard.md

```diff
@@ -0,0 +1,58 @@
+# Dashboard and Visualization
+
+## Overview
+
+Dashboard provides "at glance" view of Key Performance Index(KPI) in accrodance with goals and objectives of business processes. Typically this includes metrices and key data points to monitor the health of a business, department or specific process. Dashboard usually connects to one or multiple data sources allowing the creation of visual representation of data. This is important in observability as a good dashboard will be able to 
+
+- show trends
+- identify patterns(user, usage, search etc)
+- measure efficiency easily
+- identify data outliers and correlations
+- view health state or performance of the system
+- give an outlook of the KPIs that is important to a business/process
+
+which otherwise is difficult to understand and consume relating back to business goal or KPI measure. Common questions to ask while building dashboard would be:
+- Where did my user spend most of their time at?
+- What is my user searching?
+- How do I better help my team with alerts and troubleshooting?
+- Is my system healthy for the past one day/week/month/quarter?
+
+## Best Practices
+
+1. Identify goals or KPI measurement. Identifying goals or KPI helps in defining what needs to be achieved. Here are some examples - remote call latency, average CPU/GPU consumption per time unit.
+2. Ask questions that can help reach the defined goal or KPI. This may sound counter intuitive, the more questions asked while constructing dashboard the better the outcome will be. Questions like location, internet service provider, time of day users make requests to server would be a good start.
+3. Validate the questions. This is often done with stakeholders, sponsors, leads or even project managers.
+4. Build simple chart, have more of them. Rather than having a complicated all in one dashboard, seperate the dashboard into sections. Azure Workbooks is highly recommended with markdown support.
```

Comment:
I would suggest removing the `Azure Workbooks` reference here as this is supposed to be tech agnostic.

Also, here's a suggestion for the first part of the paragraph:
"Separate a dashboard in multiple sections for simplicity. Add multiple and simple charts. "

Reply:
Removed workbooks from best practices section